src/igvfd/schemas/prediction_set.json

```diff
@@ -244,13 +244,18 @@
         },
         "scope": {
             "title": "Prediction Scope",
-            "description": "The scope or scale that this prediction set is designed to target. If the scope is across gene(s) or loci, these will need to be specified in the genes or loci property.",
+            "description": "The scope or scale that this prediction set is designed to target.",
             "type": "string",
             "enum": [
                 "genes",
                 "loci",
                 "genome-wide"
             ],
+            "enum_descriptions": {
```

Comment:
the use of "upon" here is weird but I can't come up with anything better at the moment.